recipes/node-url-to-whatwg-url/src/url-format.ts

```diff
@@ -103,44 +109,45 @@ const getValue = (pair: SgNode<JS>): V | undefined => {
  */
 function urlFormatToUrlToString(callNode: SgNode<JS>[], edits: Edit[]): void {
 	for (const call of callNode) {
-		const optionsMatch = call.getMatch("OPTIONS");
+		const optionsMatch = call.getMatch('OPTIONS');
 		if (!optionsMatch) continue;
 
 		// Find the object node that contains the URL options
-		const objectNode = optionsMatch.find({ rule: { kind: "object" } });
+		const objectNode = optionsMatch.find({ rule: { kind: 'object' } });
 		if (!objectNode) continue;
 
 		const urlState: UrlState = {};
 
-		const pairs = objectNode.findAll({ rule: { kind: "pair" } });
+		const pairs = objectNode.findAll({ rule: { kind: 'pair' } });
 
 		for (const pair of pairs) {
-			const keyNode = pair.find({ rule: { kind: "property_identifier" } });
+			const keyNode = pair.find({ rule: { kind: 'property_identifier' } });
 			const key = keyNode?.text();
 			if (!key) continue;
 
-			if (key === "query") {
+			if (key === 'query') {
 				// Collect query object literals into key=value
-				const queryObj = pair.find({ rule: { kind: "object" } });
+				const queryObj = pair.find({ rule: { kind: 'object' } });
 				if (queryObj) {
-					const qpairs = queryObj.findAll({ rule: { kind: "pair" } });
+					const qpairs = queryObj.findAll({ rule: { kind: 'pair' } });
 					const list: Array<[string, string]> = [];
 
 					for (const qp of qpairs) {
-						const qkeyNode = qp.find({ rule: { kind: "property_identifier" } });
+						const qkeyNode = qp.find({ rule: { kind: 'property_identifier' } });
 						const qvalLiteral = getLiteralText(
 							qp.find({
 								rule: {
 									any: [
-										{ kind: "string" },
-										{ kind: "number" },
-										{ kind: "true" },
-										{ kind: "false" },
+										{ kind: 'string' },
+										{ kind: 'number' },
+										{ kind: 'true' },
+										{ kind: 'false' },
 									],
 								},
 							}),
 						);
-						if (qkeyNode && qvalLiteral !== undefined) list.push([qkeyNode.text(), qvalLiteral]);
+						if (qkeyNode && qvalLiteral !== undefined)
+							list.push([qkeyNode.text(), qvalLiteral]);
```

Comment:
I think this is not better: either keep inline or it needs `{}`

Reply:
+ 1 for `(}` since inline is +80 in collum